x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/action/EqlSpecLoader.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.action;
+
+import io.netty.util.internal.StringUtil;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EqlSpecLoader {
+    public static List<EqlSpec> load(String path, boolean supported) throws Exception {
```

Comment:
Will this method be used in the future? (atm it isn't)

Reply:
EqlSpecLoader.load is used here 
https://github.com/elastic/elasticsearch/pull/52248/files#diff-663165787e62c937a0aa8ac3d8511d6dR63